{"name": "test_change_PUT_ID_Pet", "status": "broken", "statusDetails": {"message": "RuntimeError: No active exception to reraise", "trace": "self = <tests.test_PetDemoProject_api.Test_Pets object at 0x0000022305D17B90>\n\n    @allure.description(\"Изменение имени животного по ID\")\n    def test_change_PUT_ID_Pet(self):\n>       Pet_Demo_api.put_change_Pet()\n\ntests\\test_PetDemoProject_api.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    @staticmethod\n    def put_change_Pet():\n        if Pet_Demo_api.ID is None:\n            print(\"ID не установлен\")\n>           raise  # указываем об ошибке\nE           RuntimeError: No active exception to reraise\n\nutils\\API_Pets.py:101: RuntimeError"}, "description": "Изменение имени животного по ID", "attachments": [{"name": "stdout", "source": "f33d16e4-d60e-4afe-a004-45d853323df9-attachment.txt", "type": "text/plain"}], "start": 1696768305141, "stop": 1696768305142, "uuid": "f0b71910-0be8-41cd-bc09-b4ff1ce05d9b", "historyId": "792438aa1c1647aefcfca422629b1f31", "testCaseId": "ff8cca43b452a837cb616d4a5210f513", "fullName": "tests.test_PetDemoProject_api.Test_Pets#test_change_PUT_ID_Pet", "labels": [{"name": "epic", "value": "Управление животными"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_PetDemoProject_api"}, {"name": "subSuite", "value": "Test_Pets"}, {"name": "host", "value": "LAPTOP-DQ0BFIQ6"}, {"name": "thread", "value": "12036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_PetDemoProject_api"}]}